pkgs/top-level/python-packages.nix

```diff
@@ -3677,6 +3677,24 @@ in {
     };
   };
 
+  bcolz = buildPythonPackage rec {
```

Comment:
this needs to be moved to `python-modules/bcolz/default.nix` as explained in the header of this file

Reply:
Moved.